examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>but now marginalizing over the latent <code>pi</code>  dirichlet parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>You're suggesting that I shouldn't use the latex $\pi$ when the code has the word <code>pi</code>? </p><p> </p><p>Presumably I should be consistent with this everywhere, so what are you suggesting as the overall rule for when to use which?</p><p> </p><p>Feels like there some value in a semantic separation between the <em>code</em> and the model itself, although I agree that I don't currently make that explicit in any of the text.</p>

Reply:
Sorry for being unclear. I meant "marginalizing" instead of "ignoring". It might make click what exactly means to marginalize / what is being marginalized.

Reply:
<p>Oh! Thanks for the clarification.</p>

Reply:
<blockquote>You're suggesting that I shouldn't use the latex π when the code has the word <code>pi</code>? </blockquote><p>Just saw that but yeah I agree: this is confusing for the reader, as I noted above</p>

Reply:
<p>Related to the <code>pi</code>  I was not having an issue with it, but I do agree about using the same. You can also use the π symbol in Python as a varible if you want to keep it the same in text and code.</p>

Reply:
<p>Just pushed new variable names across the board.</p><p> </p><p>Now I'm using the same variable names in the data simulation as in the model (with <code>true_</code>  prepended), and I've dropped single character symbols throughout (except for indexing and shapes). In text I use latex for all of the variable names.</p><p> </p><p>This means that I'm basically ignoring the distinction between true parameters and model parameters, which is probably the right move for this notebook.</p>